messageflux/iodevices/sqs/sqs_input_device.py

```diff
@@ -150,8 +131,8 @@ class SQSInputDeviceManager(SQSManagerBase, InputDeviceManager[SQSInputDevice]):
     SQS input device manager
     """
 
-    def __init__(self):
-        super(SQSManagerBase, self).__init__()
+    def __init__(self, sqs_resource: 'SQSServiceResource'):
```

Comment:
there are ALOT of parameters that can be changed while creating the resource... i wanted to give the user the option to init it as he wants. (otherwise, you're losing degrees of freedon

Reply:
From the experience of working with the library, it is far less convenient to configure `boto3.resource` in a pythonic way.

In all of the use cases i've experienced (working locally with a temp aws profile and initializing the boto in a k8s pod) initializing boto without args and setting (many) combinations of aws environment variables gives you all the freedom you need as a developer.

If there is a param that boto does not look for in env var, and you think is important to the configuring, then mention it explicitly.

Reply:
fixed it. now you can give it, or not(and it will create it from env vars)